azext_iot/common/_azure.py

```diff
@@ -9,6 +9,25 @@
 from azext_iot.common.utility import validate_key_value_pairs
 from azext_iot.common.utility import trim_from_start
 from azext_iot._factory import iot_hub_service_factory
+from azure.cli.core._profile import Profile
+
+
+def _get_aad_token(cmd, resource=None):
```

Comment:
Can you please add a test for this...we have unit tests for various helper methods.